srcpkgs/linux4.20/files/x86_64-dotconfig

```diff
@@ -5371,7 +5371,9 @@ CONFIG_DRM_I915=m
 CONFIG_DRM_I915_CAPTURE_ERROR=y
 CONFIG_DRM_I915_COMPRESS_ERROR=y
 CONFIG_DRM_I915_USERPTR=y
-# CONFIG_DRM_I915_GVT is not set
+CONFIG_DRM_I915_GVT=m
```

Comment:
`CONFIG_DRM_I915_GVT` is a boolean value so it should be set to `y` instead of `m`.